cypress/e2e/displays.cy.js

```diff
@@ -29,14 +29,18 @@ describe('Displays', function () {
 
         cy.visit('/display/view');
 
-        cy.get('#displays_wrapper.dataTables_wrapper').check();
+        cy.wait(500);
+
+        cy.get('#displays_wrapper.dataTables_wrapper').should('have.length', 1);
```

Comment:
Instead of checking for the element length, there's a much cleaner approach, with:
```
cy.get('#displays_wrapper.dataTables_wrapper').should('be.visible');
```